github/PaginatedList.py

```diff
@@ -153,12 +154,18 @@ def __init__(
             self.__nextParams["per_page"] = self.__requester.per_page
         self._reversed = False
         self.__totalCount: Optional[int] = None
+        self._attributesTransformer = attributesTransformer
 
         first_page = []
         if firstData is not None and firstHeaders is not None:
             first_page = self._getPage(firstData, firstHeaders)
         super().__init__(first_page)
 
+    def _transformAttribute(self, attribute: Any) -> Any:
+        if self._attributesTransformer is None:
+            return attribute
+        return self._attributesTransformer(attribute)
```

Comment:
I got the naming / semantics wrong, I think, this is more accurate:
```suggestion
    def _transformAttributes(self, element: Dict[str, Any]) -> Dict[str, Any]:
        if self._attributesTransformer is None:
            return element
        return self._attributesTransformer(element)
```